src/KawaBot.Feeder/Feeder.fs

```diff
@@ -19,3 +21,13 @@ type Feeder =
         new HttpRequestMessage(defaultArg method HttpMethod.Get, message)
         |> this._client.SendAsync
         |> Async.AwaitTask
+
+    member this.PostJSON(message: string, body: 'a) =
```

Comment:
Lets use the Pascal case for abbreviations longer than 3 chars: `JSON` -> `Json`.